### ISSUE TITLE
Monochrome "fallback" on Dynamic module not working

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#146]
On the new Dynamic module, you put in an expression and it runs it on every pixel for each color. But if you haven't specified anything for a channel, it runs the "monochrome" expression on it. This logic is a bit odd and seems to be causing some errors. 

Input `(r-b)/(r+b)*128 + 128` or even just `r` into the Dynamic module here: https://publiclab.github.io/image-sequencer/examples/#steps=dynamic

And you get a black image. That code is here, and perhaps this line needs to be multiplied by 255?

https://github.com/publiclab/image-sequencer/blob/master/src/modules/Dynamic/Module.js#L18

Or perhaps it's something more subtle, like the R, G, and B channels evaluating to 0?

**Comment by jywarren** [comment]
The contributors most likely able to help are:

@ccpandhare
@jywarren

**Comment by ccpandhare** [comment]
I am not able to reproduce your error.
Isn't this the expected result?
![screen shot](https://user-images.githubusercontent.com/20787612/32045661-774b2bb0-ba5e-11e7-91fb-0b0f18a7b64c.JPG)

**Comment by ccpandhare** [comment]
Lines 31-35 `./src/modules/Dynamic/Module.js`
```js
    channels.forEach(function(channel) {
      if (options.hasOwnProperty(channel))
            options[channel + '_function'] = generator(options[channel]);
      ...
    });
```

Shouldn't that be 
```js
    channels.forEach(function(channel) {
      if (options.hasOwnProperty(channel) && options[channel] != "")
            options[channel + '_function'] = generator(options[channel]);
      ...
    });
```

Fixed it for me!

**Comment by jywarren** [comment]
Ahh! great. Hey, if you'd like to make that change and push it to a branch with a name beginning with `first-timer-`, I think our new `first-timers-bot` will pick it up and make a FTO issue out of it! :-)))

Give it a try! New outreach technology :-)

**Comment by ccpandhare** [comment]
Ohh wow! That's interesting!

**Comment by blurry-x-face** [comment]
fixed in #1451
